packages/omg-js-childchain/src/index.js

```diff
@@ -22,49 +21,13 @@ class ChildChain {
     this.watcherUrl = watcherUrl
     this.childChainUrl = childChainUrl
   }
-  /**
```

Comment:
We might need to keep this for another demo project unfortunately.. :( But I'm totally okay with labeling this as `@Depracated`